pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -343,6 +357,14 @@ func (p *awsSDKProvider) Autoscaling(regionName string) (ASG, error) {
 	return client, nil
 }
 
+func (p *awsSDKProvider) Route53(regionName string) (R53, error) {
+	r53Client := route53.New(session.New(&aws.Config{
+		Region:      &regionName,
+		Credentials: p.creds,
+	}))
+	return r53Client, nil
```

Comment:
Let's either include the call to `addHandlers` or have a note as to why not...

.. I realize the same applies to Metadata, but that's a separate issue ;-)